intro-to-object-oriented-programming/defining-classes.md

```diff
@@ -231,10 +231,10 @@ All attributes have names. The values of attributes will differ between instance
 
 Here are some examples of attributes that could exist on classes:
 
-| Class  | Attribute                    | Instance Example                                                                      | Value of Attribute for Example |
```

Comment:
Just wanted to put the attribute column next to the attribute value column to make the attribute value column more understandable